frontend/src/views/Scribble.vue

```diff
@@ -1,39 +1,41 @@
 <template>
   <div id="scribble">
-    <Canvas/>
-    <Gameroom :clients="clients"/>
+    <CanvasPanel :colors="colors" :sizes="sizes" />
+    <Gameroom :clients="clients" />
   </div>
 </template>
 
 <script>
-import Canvas from '../components/Canvas.vue'
-import Gameroom from '../components/Gameroom.vue'
-import { EventBus } from '../eventBus.js';
-import { ArcadeWebSocket } from '../webSocket.js';
+import CanvasPanel from "../components/CanvasPanel.vue";
+import Gameroom from "../components/Gameroom.vue";
+import { EventBus } from "../eventBus.js";
+import { ArcadeWebSocket } from "../webSocket.js";
 
 export default {
-  name: 'Scribble',
+  name: "Scribble",
   data: function() {
     return {
-      clients: []
-    }
+      clients: [],
+      colors: ["#000000", "#4287f5", "#da42f5", "#7ef542"],
+      sizes: [8, 16, 32, 64]
```

Comment:
freaken lol, was looking for these values forever since like these values were passed down like 4 levels deep xD. but nice nice, I like it.